pkg/container/docker_build.go

```diff
@@ -50,7 +50,8 @@ func NewDockerBuildExecutor(input NewDockerBuildExecutorInput) common.Executor {
 		if input.Container != nil {
 			buildContext, err = input.Container.GetContainerArchive(ctx, input.ContextDir+"/.")
 		} else {
-			buildContext, err = createBuildContext(ctx, input.ContextDir, "Dockerfile")
+			buildContext, err = createBuildContext(ctx, input.ContextDir, input.Dockerfile)
+			options.Dockerfile = input.Dockerfile
```

Comment:
I only set it here because I wasn't sure what the expectation was when the `input.Container` was not nil. It would seem cleaner to set it in the above block, but I was worried about changing existing behavior.